MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -2633,234 +2633,187 @@ public GetBuildInfoResponse GetBuildInfoMultiple (WebServiceLogin login, string
 
 				lanes = db.GetAllLanes ();
 
-				switch (response.Host.QueueManagement) {
-				case DBQueueManagement.OneRevisionWorkAtATime:
```

Comment:
I didn't add a new code because no lane in private wrench uses anything but the default, the setting has a note next to it on the edit host page saying that it is ignored (which is incorrect for `OneRevisionWorkAtATime`), and this assigner should be better than the default one anyway.